packages/ai-jsx/src/batteries/constrained-output.tsx

```diff
@@ -81,13 +82,14 @@ export async function* JsonChatCompletion(
   props: Omit<TypedObjectCompletionWithRetry, 'typeName' | 'parser' | 'partialResultCleaner'>,
   { render }: AI.ComponentContext
 ) {
+  const partialResultCleaner = 'default' in untruncateJson ? untruncateJson.default : untruncateJson;
```

Comment:
Why is this `'default' in` check necessary?

Reply:
Bummer. Ok, that's fine. It would be nice to add a comment in the code saying something like this. :smile:

Reply:
Also, we'll need to upstream this change to #175.